pfunk/__main__.py

```diff
@@ -52,32 +54,36 @@ def run_named_plot(name, show=False):
     """
     Runs the plot ``name``.
     """
-    import pfunk.plots
     try:
-        pfunk.plots.run(name, show=show)
+        pfunk.tests.plot(name, show)
     except KeyError:
-        print('Plot not found: ' + name)
-        print('Available plots:')
-        for plot in pfunk.plots.plots():
-            print('  ' + plot)
+        print_avail_tests(name)
 
 
 def run_all_plots(show=False):
     """
     Runs all plots.
     """
-    import pfunk.plots
-    for name in pfunk.plots.plots():
-        pfunk.plots.run(name, show=show)
+    for name in pfunk.tests.tests():
+        pfunk.tests.plot(name, show)
 
 
-def show_plot_list():
+def run_named_analyse(name, show=False):
```

Comment:
run_named_analysis ?

Reply:
Or maybe `analyse_named_test` actually, and then `run_named_test` and `plot_named_test` (or some variant thereof)

Reply:
I like `analyse_named_test`